### PR TITLE
Add user surveys data privacy page

### LINKED ISSUE #4666: Update data-privacy page
Further to my last email, I would also like to make an amendment to the 
Privacy Notice page on our website here: 
https://www.ubuntu.com/legal/data-privacy#privacy-notices

I have put the amendment into a copydoc: 
https://docs.google.com/document/d/1axjXD2S_-DhB2FG2yEItz4cu3Cms9WZxWTaLayKcOwI/edit

Please let me know if there is anything you need from me.

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- **Asking legal if this shouldn't be made more generic**
- Add MAAS user survey data privacy page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/legal/data-privacy/snap-store](http://0.0.0.0:8001/legal/data-privacy/snap-store)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1XKWK3VvJe0Z_soJqO3F0BrrXwlP9ZObYYKxA1hym7QU/edit)

## Issue / Card

Fixes #4666

### CODE REVIEW

File: templates/legal/data-privacy/maas-user-survey.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+When you submit live feedback on the MAAS webpages, we invite you to leave your your email address and related information via Usabilla’s software platform. You may be shown different surveys depending on user features and usage. Usabilla will share this information with us.
+
+## Why do we collect this information?
+
+We will use your information for MAAS user research. We ask for your consent to do this and will only contact you for this purpose for as long as you continue to consent. This may involve inviting you to talk to us about how you use MAAS and to test new MAAS features.
+
+## What do we do with your information?
+
+Your information is stored in our database and may be processed outside of the European Economic Area. It is shared with the third parties only as reasonably required for Canonical to process and store your data in accordance with this notice.
```
Comment (anthonydillon) [inline templates/legal/data-privacy/maas-user-survey.md]: The data is not stored in our database. We just have access to the data held by Usabilla.

File: templates/legal/data-privacy/maas-user-survey.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+Such countries do not have the same data protection laws as the United Kingdom and EEA. Where the European Commission has not given a formal decision that such countries provide an adequate level of data protection similar to those which apply to the United Kingdom and EEA, any transfer of your personal information will be subject to approved contract terms designed to help safeguard your privacy rights and give you remedies in the unlikely event of misuse of you personal information. If you would like further information please contact us (see “Your right to complain” below). We will not otherwise transfer your personal data outside of the EEA.
+
+## How long do we keep your information for?
+
+We keep your information for as long as you continue to consent to participate in MAAS user research and thereafter for so long as reasonably required in accordance with our record retention policy.
```
Comment (anthonydillon) [inline templates/legal/data-privacy/maas-user-survey.md]: We do not actually store or hold any user information.

File: templates/legal/data-privacy/maas-user-survey.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+- object to decisions being taken by automated means which produce legal effects concerning you or similarly significantly affect you;
+- object in certain other situations to our continued processing of your personal information;
+- otherwise restrict our processing of your personal information in certain circumstances;
+- claim compensation for damages caused by out breach of any data protection laws.
+
+For further information on each of those rights, including the circumstances in which they apply, see the Guidance from the UK Information Commissioner’s Office (ICO) on individuals rights under the General Data Protection Regulation.
+
+If you would like to exercise any of those rights, please:
+
+- email, call or write to us, please see “Your right to complain” section below;
```
Comment (anthonydillon) [inline templates/legal/data-privacy/maas-user-survey.md]: We do not provide a phone number to call.

**Comment by anasereijo** [comment]
one tiny thing:

privacy notice snap store ubuntu

**Comment by anasereijo** [comment]
LGTM 👍

**Pull request merged**